packages/web-components/src/components/va-language-toggle/va-language-toggle.tsx

```diff
@@ -22,7 +22,7 @@ export class VaLanguageToggle {
   /**
    * The ISO language code for the page. Default is 'en'.
    */
-  @Prop() language: string = 'en';
+  @Prop({ mutable: true }) language: string = 'en';
```

Comment:
Making this prop mutable resolves this Stencil warning.

![Screenshot 2024-11-19 at 11 16 51 AM](https://github.com/user-attachments/assets/e20a2023-3109-4459-a637-b61b7c6024c5)